src/momento/aio/simple_cache_client.py

```diff
@@ -289,7 +305,15 @@ async def delete(self, cache_name: str, key: str) -> CacheDeleteResponse:
             AuthenticationError: If the provided Momento Auth Token is invalid.
             InternalServerError: If server encountered an unknown error while trying to delete the item.
         """
-        return await self._data_client.delete(cache_name, key)
+        return await self._get_next_client().delete(cache_name, key)
+
+    def _get_next_client(self) -> _ScsDataClient:
+        # print(f"Using client {self._next_client_index}")
```

Comment:
mean to leave this? Maybe should be a debug log statement?

Reply:
nope thanks will get rid of it

Reply:
fixed in https://github.com/momentohq/client-sdk-python/pull/125/commits/ed394103eccc9848fb371fc800baab9805d2c935